data/ecosystem/vendors.yaml

```diff
@@ -254,7 +254,7 @@
 - name: Sumo Logic
   distribution: true
   nativeOTLP: true
-  url: 'https://help.sumologic.com/docs/apm/traces/quickstart/'
+  url: 'https://help.sumologic.com/docs/send-data/opentelemetry-collector/'
   contact: ''
```

Comment:
While updating the entry can you also provide a contact here. Can be your GitHub handle (or someone else’s) or an email address we can use to reach out to in case of issues (dead links, etc.)

Reply:
Sure, I've added the link to my GitHub profile page, which also has my email address.